serverless.yml

```diff
@@ -124,6 +127,19 @@ stepFunctions:
                 - States.ALL
                 Next: errorHandler
             Next: fieldVisitTransform
+          tsFieldVisitReadings:
+            Type: Task
+            Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-capture-field-visit-metadata${self:provider.stage}-preProcess
+            Parameters:
+              id.$: $.id
+              partitionNumber.$: $.partitionNumber
+            ResultPath: $.tsFieldVisitResult
```

Comment:
Let's call this something like `tsFieldVisitReadingsResult` to differentiate it from the `ResultPath` in the `tsFieldVisit` state a few lines above.  This will ensure that the result values coming out of this lambda won't collide with those of the other state.